src/instana/agent/host.py

```diff
@@ -176,7 +176,10 @@ def announce(self, discovery):
             url = self.__discovery_url()
             response = self.client.put(url,
                                        data=to_json(discovery),
-                                       headers={"Content-Type": "application/json"},
+                                       headers={
```

Comment:
These exact four lines are repeated 4 ish times, 5 if we count paritals. Perhaps it makes sense to use a variable?